corehq/apps/userreports/templates/userreports/edit_report_config.html

```diff
@@ -27,23 +27,45 @@
     </div>
   </div>
   <div class="clearfix"></div>
-  {% crispy form %}
-  <br/>
-  {% if not referring_apps  %}
-    <label class="label label-danger">This report is not referred by any apps in this domain. Please note, a static report might be referred by apps in other domains.</label>
-  {% else %}
-    <fieldset>
-      <legend>Apps dependent on this report</legend>
-      <ul>
-        {% for app in referring_apps %}
-          <li>
-            <a href="{{ app.module_url }}">{{ app.module_name}}</a> module in
-            <a href="{{ app.app_url }}">{{app.app_name}}</a> app
-          </li>
-        {% endfor %}
-      </ul>
-    </fieldset>
-  {% endif %}
+
+  <ul class="nav nav-tabs">
+    <li class="active"><a data-toggle="tab" href="#tabs-configuration">Configuration</a></li>
```

Comment:
just curious, is `#tab-container-name` the preferred way for this? i would have just used `configuration` here.

Reply:
I like using more specific names for ids, for fear of clashing with something else on the page (if this page got big like module view, I could imagine another area using `configuration`) but we don't have specific guidelines, it's just a preference.

Reply:
thank you